sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -69,7 +70,7 @@ Install the Azure Text Analytics client library for Python with [pip][pip]:
 pip install azure-ai-textanalytics --pre
 ```
 
-> Note: This version of the client library defaults to the v3.1-preview version of the service
+> Note: This version of the client library defaults to the v3.1-preview.2 version of the service
```

Comment:
Is not the default the latest?
```suggestion
> Note: This version of the client library defaults to the v3.1-preview.3 version of the service
```

Reply:
The reason we can't use v3.1-preview.3 for all of the other services is because the other services are not all using `/v3.1-preview.3` in their controller paths.  I tried doing it this way originally (setting v3.1-preview.3 as the default) but later faced issues in existing tests for other services.

Reply:
what do you mean by other services? other APIs? I am running the old JS testsuite with `preview.3` just fine.

Reply:
Yes, I meant other APIs.  Maybe this is something to do with the way things are set up specifically in the Python SDK.  Basically what is happening is that if I use the autogenerated code from v3.1-preview.3 for the other services I get errors because all of the autogenerated methods have an associated endpoint which have the API version in the path.  

It's possible I may have done something wrong in the configuration in regards to the API version.  I'll check with @iscai-msft to see if this can be easily resolved (this is also kind of under discussion in another comment below).

Reply:
I don't think you generated things incorrectly, my understanding with @abhahn was that certain endpoints (i.e. sentiment) weren't deployed in service v3.1-preview.3 (at least at the time of release), @abhahn to confirm. @abhahn also there should always be the API version in the path, that's the template for TA endpoints, so this should be good. @abhahn do you think the other endpoints could now be deployed to preview 3, which is why the js pipeline works?